tests/test_models_up_to_date.py

```diff
@@ -0,0 +1,88 @@
+"""Test if models are up to date."""
```

Comment:
Nice idea to add these tests! :+1:

Reply:
Indeed, they are our safety net to ensure our models and generated code are in sync.